src/views/pages/address/choose-address-form.njk

```diff
@@ -22,25 +22,33 @@
         name: "selected_address",
         value: cache | dump,
         fieldset: {
-        legend: {
+          legend: {
             text: pageTitle,
             isPageHeading: true,
             classes: "govuk-fieldset__legend--xl"
-        }
+          }
+        },
+        hint: {
+          text: pageTitleHint
         },
         items: addressItems
     }) }}
 
+    <p>
+      <a href="{{ props.currentUrl | replace(props.pageType, props.data.enterManualAddressPageType) }}" class="govuk-body-m govuk-link" data-event-id="address-not-on-list">{{ addressLink }}</a>
+    </p>
+
+    {% if publicRegisterInformationHeading %}
+      {% include "includes/public-register-information.njk" %}
+    {% endif %}
```

Comment:
made the public register text optional for now, i'm assuming we'll have to add it retrospectively to the other choose address pages

Reply:
Will you create a ticket for this or let Vern know?

Reply:
I'll create a ticket